### PR TITLE
doc: Add historical release notes for 0.7.0

### PR DESCRIPTION
**Pull request by peterwillcn**
Signed-off-by: xiaobo <peterwillcn@gmail.com>

### CODE REVIEW

File: docs/release-notes/release-notes-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+As previously-supported CPU platforms, this release's pre-compiled
+distribution provides binaries for the x86_64 platform.
+
+Notable changes
+===============
+
+Support CR Penalty.
+Support Revert to Pow.
+Support Cross-chain Small Quick Transfers.
```
Comment (RainFallsSilent) [inline docs/release-notes/release-notes-0.7.0.md:37]: Not support

File: docs/release-notes/release-notes-0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Support Random DPOS Node.
+
```
Comment (RainFallsSilent) [inline docs/release-notes/release-notes-0.7.0.md:39]: remove blank line

File: docs/release-notes/release-notes-0.7.0.md (hunk 1 above)
Comment (RainFallsSilent) [reply docs/release-notes/release-notes-0.7.0.md]: fixed

File: docs/release-notes/release-notes-0.7.0.md (hunk 2 above)
Comment (RainFallsSilent) [reply docs/release-notes/release-notes-0.7.0.md]: fixed

**Review by RainFallsSilent — APPROVED** [review]

**Review by Houshoupei84 — APPROVED** [review]

**Review by xiaomingloveqinger — APPROVED** [review]

**Review by bocheng0000 — APPROVED** [review]

**Pull request merged**